lib/matplotlib/axes/_axes.py

```diff
@@ -6871,6 +6880,15 @@ def hist2d(self, x, y, bins=10, range=None, normed=False, weights=None,
           `.colors.PowerNorm`.
         """
 
+        if density is not None and normed is not None:
+            raise ValueError("kwargs 'density' and 'normed' cannot be used "
+                             "simultaneously. Please only use 'density', "
+                             "since 'normed' is deprecated.")
+        if normed is not None:
+            cbook.warn_deprecated("2.1", name="'normed'", obj_type="kwarg",
```

Comment:
You cannot deprecate for "2.1". The deprecation has to be always for the next point release, i.e. currently "3.1". `removal` can be omitted. That's two minor point releases by default.